lib/db.js

```diff
@@ -302,6 +312,61 @@ class LimitDBRedis extends EventEmitter {
       });
   }
 
+  buildStatusResponse(bucket, prefixLength) {
+    return (key, lastDrip, remaining) => {
+      const bucketKeyConfig = this.bucketKeyConfig(bucket, key);
+      return {
+        key: key.substring(prefixLength),
+        limit: bucketKeyConfig.size,
+        reset: this.calculateReset(bucketKeyConfig, remaining),
+        remaining
+      };
+    };
+  }
+
+  /**
+   * Find all buckets that match the key.
+   *
+   * @param {statusParams} params
+   * @param {function(Error, statusResult)} [callback]
+   */
+  status(params, callback) {
+    const valError = this.validateParams(params);
+    if (valError) {
+      return setImmediate(callback, valError);
+    }
+    const bucket = this.buckets[params.type];
+    let prefix = `${params.type}:${params.key}*`;
+    let prefixLength = `${params.type}:`.length;
+
+    // Calling nodes directly does not use the global prefix
+    if (this.prefix) {
+      prefix = `${this.prefix}${prefix}`;
+      prefixLength = `${this.prefix}${params.type}:`.length
+    }
+
+    const count = params.count || 50;
```

Comment:
So count is not really the max as it says above, in fact it is the max per shard

Reply:
True, will adjust.